test/containerized_test.py

```diff
@@ -16,13 +25,113 @@
 from . import http
 
 
-TICKET_SIZE = 16 * KiB
+log = logging.getLogger("test")
+
+FILE_SIZE = 16 * KiB
+
+SERVER_PORT = 8080
+CONTAINER_IMAGE = "localhost/ovirt-imageio:latest"
+
+
+class ContainerPath:
+
+    def __init__(self, file_name, folder):
+        self.file_name = file_name
+        self.folder = folder
```

Comment:
Wanted to avoid having too many repetitions of the `os.path.dirname()`. And actually looking for a more readable option. But in the current version I opted for global variables.